src/Native/gc/unix/configure.cmake

```diff
@@ -95,4 +95,4 @@ endif()
 
 check_library_exists(${PTHREAD_LIBRARY} pthread_getaffinity_np "" HAVE_PTHREAD_GETAFFINITY_NP)
 
-configure_file(${CMAKE_CURRENT_LIST_DIR}/config.h.in ${CMAKE_CURRENT_BINARY_DIR}/config.h)
+configure_file(${CMAKE_CURRENT_LIST_DIR}/unixconfig.h.in ${CMAKE_CURRENT_BINARY_DIR}/unixconfig.h)
```

Comment:
I've renamed this file because the unix pal defines a file with the same name

Reply:
I would be better to have gc in the name of this file to make it clear that it is for gc and to avoid future conflicts. E.g. `config.gc.h` .

Reply:
Nice! i was out of ideas after `gcconfig.h` was taken :+1: